pkg/utils/webhook/add.go

```diff
@@ -44,6 +44,8 @@ type Options struct {
 	ServiceNamespace string
 	// port of the service
 	ServicePort int32
+	// external service URL
+	ServiceURL string
```

Comment:
I think we should keep the naming and call it `WebhhokURL` otherwise the change from `webhookURL` int eh options to `serviceURL` in the webhook confuses

Reply:
Thank you. Done.

Reply:
Thank you for the review.
I have changed it to `WebhookURL`.